test/micro/org/openjdk/bench/java/lang/foreign/libPtr.c

```diff
@@ -29,16 +29,18 @@
 #define EXPORT
 #endif
 
+#include <stddef.h>
+
 EXPORT long long id_long_long(long long value) {
   return value;
 }
 
 EXPORT long long id_ptr_long(void* ptr) {
-  return (long long)ptr;
+  return (long long)(size_t)ptr;
```

Comment:
Note - these benchmarks do not care about the fact that some bits in the original value could be dropped to the floor. The benchmark aims at comparing the performance of passing/returning longs with that of passing/returning segments (e.g. to make sure that escape analysis works its magic). As such, lossy conversions are immaterial here.

Reply:
I guess a different approach could be to avoid using `void*` and use `size_t` instead, which would still allow FFM to link correctly. If that is preferred I can do that instead.

Reply:
I think this is a good solution.